react-carousel-component/my-app/src/IndexButtons.js

```diff
@@ -0,0 +1,19 @@
+import './IndexButtons.css';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCircle as solid } from '@fortawesome/free-solid-svg-icons';
+import { faCircle as regular } from '@fortawesome/free-regular-svg-icons';
+
+export default function IndexButtons({ items, onIndex, index }) {
+  return (
+    <div>
+      {items.map((item) => (
+        <FontAwesomeIcon
+          icon={index === items.indexOf(item) ? solid : regular}
+          key={item}
```

Comment:
If you have duplicate items this also will result in duplicate keys. The index would be better here, as well.